# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jsaw/gnomesword/glib-h-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN gnomesword-2.1.1/src/backend/config.cc gnomesword-2.1.1-jsaw/src/backend/config.cc
--- gnomesword-2.1.1/src/backend/config.cc	2004-05-15 17:02:56.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/backend/config.cc	2004-08-09 03:52:23.906843024 +0200
@@ -43,7 +43,7 @@
 #include <stdio.h>
 #include <sys/stat.h>
  
-#include <glib-2.0/glib.h>
+#include <glib.h>
 
 #include "config.hh"
 #include "sword_main.hh"
diff -urN gnomesword-2.1.1/src/backend/config.hh gnomesword-2.1.1-jsaw/src/backend/config.hh
--- gnomesword-2.1.1/src/backend/config.hh	2004-03-18 21:45:40.000000000 +0100
+++ gnomesword-2.1.1-jsaw/src/backend/config.hh	2004-08-09 03:52:23.908842720 +0200
@@ -25,7 +25,7 @@
 #ifdef __cplusplus
 extern "C" {
 #endif	
-#include <glib-2.0/glib.h>
+#include <glib.h>
 //#include "main/sword.h"
 
 int backend_open_config_file(char *file);
diff -urN gnomesword-2.1.1/src/backend/module_manager.cc gnomesword-2.1.1-jsaw/src/backend/module_manager.cc
--- gnomesword-2.1.1/src/backend/module_manager.cc	2004-05-15 16:53:00.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/backend/module_manager.cc	2004-08-09 03:52:23.914841808 +0200
@@ -31,7 +31,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #include <gnome.h>
 #else
 #include <glib-1.2/glib.h>
diff -urN gnomesword-2.1.1/src/backend/sword.cpp gnomesword-2.1.1-jsaw/src/backend/sword.cpp
--- gnomesword-2.1.1/src/backend/sword.cpp	2004-07-14 16:31:50.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/backend/sword.cpp	2004-08-09 03:52:23.917841352 +0200
@@ -31,7 +31,7 @@
 #endif
 
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #include <gnome.h>
 #include <swmgr.h>
 #include <markupfiltmgr.h>
diff -urN gnomesword-2.1.1/src/main/bibletext.c gnomesword-2.1.1-jsaw/src/main/bibletext.c
--- gnomesword-2.1.1/src/main/bibletext.c	2004-05-15 16:40:42.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/bibletext.c	2004-08-09 03:52:23.920840896 +0200
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword-2.1.1/src/main/commentary.cc gnomesword-2.1.1-jsaw/src/main/commentary.cc
--- gnomesword-2.1.1/src/main/commentary.cc	2004-04-21 22:57:50.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/commentary.cc	2004-08-09 03:52:23.923840440 +0200
@@ -27,7 +27,7 @@
 #include <swmodule.h>
 #include <swconfig.h>
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 
 #include "main/commentary.h"
 #include "main/settings.h"
diff -urN gnomesword-2.1.1/src/main/configs.c gnomesword-2.1.1-jsaw/src/main/configs.c
--- gnomesword-2.1.1/src/main/configs.c	2003-09-03 19:52:54.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/configs.c	2004-08-09 03:52:23.928839680 +0200
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword-2.1.1/src/main/configs.h gnomesword-2.1.1-jsaw/src/main/configs.h
--- gnomesword-2.1.1/src/main/configs.h	2004-03-18 21:45:41.000000000 +0100
+++ gnomesword-2.1.1-jsaw/src/main/configs.h	2004-08-09 03:52:23.930839376 +0200
@@ -25,7 +25,7 @@
 #ifdef __cplusplus
 extern "C" {
 #endif	
-#include <glib-2.0/glib.h>
+#include <glib.h>
 	
 typedef struct _mod_font MOD_FONT;
 struct  _mod_font {
diff -urN gnomesword-2.1.1/src/main/dictlex.c gnomesword-2.1.1-jsaw/src/main/dictlex.c
--- gnomesword-2.1.1/src/main/dictlex.c	2004-04-21 22:57:50.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/dictlex.c	2004-08-09 03:52:23.932839072 +0200
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword-2.1.1/src/main/lists.cc gnomesword-2.1.1-jsaw/src/main/lists.cc
--- gnomesword-2.1.1/src/main/lists.cc	2004-05-15 16:53:00.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/lists.cc	2004-08-09 03:52:23.936838464 +0200
@@ -24,7 +24,7 @@
 #endif
 
 #include <gnome.h>
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #include <gal/widgets/e-unicode.h>
 
 #include <string.h>
diff -urN gnomesword-2.1.1/src/main/lists.h gnomesword-2.1.1-jsaw/src/main/lists.h
--- gnomesword-2.1.1/src/main/lists.h	2004-03-18 21:45:41.000000000 +0100
+++ gnomesword-2.1.1-jsaw/src/main/lists.h	2004-08-09 03:52:23.938838160 +0200
@@ -27,7 +27,7 @@
 {
 #endif	
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 
 typedef struct _module_lists MOD_LISTS;
 struct _module_lists {
diff -urN gnomesword-2.1.1/src/main/mod_mgr.c gnomesword-2.1.1-jsaw/src/main/mod_mgr.c
--- gnomesword-2.1.1/src/main/mod_mgr.c	2004-05-14 22:27:44.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/mod_mgr.c	2004-08-09 03:52:23.942837552 +0200
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #include <gnome.h>
 #else
 #include <glib-1.2/glib.h>
diff -urN gnomesword-2.1.1/src/main/percomm.c gnomesword-2.1.1-jsaw/src/main/percomm.c
--- gnomesword-2.1.1/src/main/percomm.c	2004-04-21 22:57:56.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/percomm.c	2004-08-09 03:52:23.945837096 +0200
@@ -25,7 +25,7 @@
 
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword-2.1.1/src/main/search_dialog.h gnomesword-2.1.1-jsaw/src/main/search_dialog.h
--- gnomesword-2.1.1/src/main/search_dialog.h	2004-05-13 21:48:27.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/search_dialog.h	2004-08-09 03:52:23.947836792 +0200
@@ -26,7 +26,7 @@
 extern "C" {
 #endif
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 
 void main_dialog_search_percent_update(char percent, void *userData);
 void main_range_text_changed(GtkEditable * editable);	
diff -urN gnomesword-2.1.1/src/main/search_sidebar.h gnomesword-2.1.1-jsaw/src/main/search_sidebar.h
--- gnomesword-2.1.1/src/main/search_sidebar.h	2004-04-21 22:58:01.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/search_sidebar.h	2004-08-09 03:52:23.949836488 +0200
@@ -26,7 +26,7 @@
 extern "C" {
 #endif
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 	
 void main_sidebar_search_percent_update(char percent, void *userData);
 void main_do_sidebar_search(gpointer user_data);
diff -urN gnomesword-2.1.1/src/main/settings.c gnomesword-2.1.1-jsaw/src/main/settings.c
--- gnomesword-2.1.1/src/main/settings.c	2004-05-18 02:24:17.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/settings.c	2004-08-09 03:52:23.955835576 +0200
@@ -25,7 +25,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword-2.1.1/src/main/spell_pspell.c gnomesword-2.1.1-jsaw/src/main/spell_pspell.c
--- gnomesword-2.1.1/src/main/spell_pspell.c	2003-12-08 16:43:55.000000000 +0100
+++ gnomesword-2.1.1-jsaw/src/main/spell_pspell.c	2004-08-09 03:52:23.958835120 +0200
@@ -26,7 +26,7 @@
 #ifdef USE_PSPELL
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword-2.1.1/src/main/xml.c gnomesword-2.1.1-jsaw/src/main/xml.c
--- gnomesword-2.1.1/src/main/xml.c	2004-05-13 17:30:33.000000000 +0200
+++ gnomesword-2.1.1-jsaw/src/main/xml.c	2004-08-09 03:52:23.968833600 +0200
@@ -24,7 +24,7 @@
 #endif
 
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #include <libxml/parser.h>
 
 
